pom.xml

```diff
@@ -220,6 +220,27 @@
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-pmd-plugin</artifactId>
+                    <configuration>
+                        <excludeRoots>
+                            <!-- exclude the generated code from PMD, Findbugs will still run -->
+                            <excludeRoot>${project.build.directory}/generated-sources/swagger-codegen</excludeRoot>
+                        </excludeRoots>
```

Comment:
Previous config was inherited, actual change removes these lines:
```xml
 <!-- generated resources -->
                            <exclude>**/com/okta/sdk/resource/**/*.java</exclude>
                            <exclude>**/com/okta/sdk/client/Client.java</exclude>
<exclude>**/com/okta/sdk/impl/resource/Default*.java</exclude>
<exclude>**/com/okta/sdk/impl/resource/**/Default*.java</exclude>
<exclude>**/com/okta/sdk/impl/client/DefaultClient.java</exclude>
```
And replaces them with the `excludeRoots` config